src/handlers/multiSig/multiSigExecuted/multiSigExecuted.ts

```diff
@@ -0,0 +1,58 @@
+import { getChainId } from '~provider';
+
+import { ContractEvent } from '~types';
+import { getMultiSigClient } from '~utils';
+// import { linkPendingMetadata } from '~utils/colonyMetadata';
+
+import {
+  getMultiSigDatabaseId,
+  getMultiSigFromDB,
+  updateMultiSigInDB,
+} from '../helpers';
+
+export default async (event: ContractEvent): Promise<void> => {
+  const {
+    colonyAddress,
+    args: { motionId, success },
+    timestamp,
+  } = event;
+
+  if (!colonyAddress) {
+    return;
+  }
+
+  const multiSigClient = await getMultiSigClient(colonyAddress);
+
+  if (!multiSigClient) {
+    return;
+  }
+
+  const chainId = getChainId();
+  const multiSigDatabaseId = getMultiSigDatabaseId(
+    chainId,
+    multiSigClient.address,
+    motionId,
+  );
+  const finalizedMultiSig = await getMultiSigFromDB(multiSigDatabaseId);
+
+  if (finalizedMultiSig) {
+    /* @TODO fix this up when we start porting over domain motions
+     * the action string is not an event argument like it's for motions, so we may need to tweak linkPendingMetadata a bit
+    await linkPendingMetadata(
+      action,
+      colonyAddress,
+      finalizedMultiSig.id,
+      true,
+    );
+    */
```

Comment:
Forgot to comment on this one yesterday, you can potentially get it from contracts or even better, store it in the DB when multisig gets created

Reply:
thanks for this, I'll copy this comment to an issue which will tackle this. Sadly I currently do not have any way of testing if it will work because those motions do not have multi sig support yet